examples/ts-plugin-with-custom-options.mjs

```diff
@@ -0,0 +1,8 @@
+// Module code is always strict mode code.
+// http://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code
+
+const plugin = function (fastify, options, next) {
```

Comment:
I would make this an async named function instead

Reply:
Fixed both in the example and the README